packages/error-reporting/src/interfaces/manual.js

```diff
@@ -74,6 +74,10 @@ function handlerSetup(client, config) {
     }
 
     if (err instanceof ErrorMessage) {
+      var fauxError = new Error(err.message);
```

Comment:
Are these error message objects constructed by the user (could they be queued before publishing)? We want to make sure that the stack reflects the accurate stack where the error was captured if possible.